downloader/src/funTest/kotlin/CvsWorkingTreeTest.kt

```diff
@@ -31,12 +31,12 @@ import io.kotlintest.specs.StringSpec
 
 import java.io.File
 
-class CvsTest : StringSpec() {
+class CvsWorkingTreeTest : StringSpec() {
```

Comment:
commit-msg: Did you mean to say `lengthy` instead of `length`?

Reply:
Yes, thanks, fixed.